src/amchart/Column.js

```diff
@@ -51,35 +58,9 @@
             this._chart.valueAxes[0].stackType = "none";
         }
 
-        // Color Palette
-        switch(this.paletteGrouping()) {
-            case "By Category":
-                this._chart.dataProvider.forEach(function(dataPoint,i){
-                    context._chart.dataProvider[i].color = context._palette(i);
-                    context._chart.dataProvider[i].linecolor = context.lineColor() !== null ? context.lineColor() : context._palette(i);
-                });
-                this._chart.colors = [];
-            break;
-            case "By Column":
-                /* falls through */
-            default:
-                this._chart.colors = this.columns().filter(function (d, i) { return i > 0; }).map(function (row) {
-                    return this._palette(row);
-                }, this);
-            break;
-        }
-
         this._chart.depth3D = this.Depth3D();
         this._chart.angle = this.Angle3D();
-        this._chart.categoryAxis.startOnAxis = false; //override due to render issue
-
-        if (this._rangeType === "candle-ohlc") {
-            this._gType = this.useOhlcLines() ? "ohlc" : "candlestick";
-        } else {
-            this._gType = "column";
-        }
-
-        this.buildGraphs(this._gType);
+        this._chart.categoryAxis.startOnAxis = false; //override due to render issue ?? does this stole hold true?
```

Comment:
Does this need a re-test to see if its needed?  If so comment should be updated.